src/components/common/client-info/types.ts

```diff
@@ -17,6 +17,7 @@ export interface ClientInfoServiceInterface {
   clientInfo: (
     sessionId: string,
     clientSessionId: string,
-    sourceIp: string
+    sourceIp: string,
+    persistentSessionId: string
```

Comment:
Does this need to be marked as optional for the types as it might not always exist? `persistentSessionId?: string`